templates/gym-ai-coach-agent/README.md

```
@@ -0,0 +1,25 @@
+# Gym AI Coach Agent
+
+## Problem
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)